Update the information about the neurodiversity items available (#795)
EuroPython/website: src/content/pages/accessibility.mdx

### Low stimulation room (South room 243)

> - If you feel overwhelmed and need a quiet space with no distraction feel free to visit **South Room 243** on the **2nd floor**. To get there look for childcare and keep on going through the hallway. It will be one of the doors to your left.

Reviewer: If you feel overwhelmed and need a quiet space with no distractions, feel free to visit **South Room 243** on the **2nd floor**. To get there, look for the childcare room and continue through the hallway. It will be one of the doors to your left.

> - You'll find a bag with sunglasses, eyemasks and earplugs for your convenience there. Please return the sunglasses and eyemasks back. Earplugs are yours forever.

Reviewer: ```suggestion
- In this room, you'll find a bag with sunglasses, eye masks, and earplugs for your convenience. Please return the sunglasses and eye masks after use. The earplugs are yours to keep.
```

### Neurodiversity bag

> We know that some conference participants can feel overwhelmed by large crowds. To help with this, we've prepared a bag full of squishy and plushy items for you to relax during the demanding conference days.

Reviewer: Some people might feel that we don't really "know" their experience but appreciate that our understanding. 
```suggestion
We understand that some conference participants can feel overwhelmed by large crowds. To help with this, we've prepared a bag full of squishy and plushy items for you to relax during the demanding conference days.
```

> It is placed at the Info desk in the Registration area. Feel free to play with anything you like for as long as you need and please bring it back for others to enjoy as well.

Reviewer: My suggestion is to make it into a wee list to make it easier to read.
```suggestion
- Location: The bag is placed at the Info Desk in the Registration area.
```

> The bag contains among others fidget, squishy and pop it toys.

Reviewer: ```suggestion
- Contents: The bag contains various items, including fidget toys, squishy toys, and pop-it toys.
- Usage: Feel free to play and stim with anything you like for as long as you need. We kindly ask that you bring the items back so others can enjoy them as well.
```
